script/playgroundHelpers.sh

```
@@ -862,6 +862,548 @@ function syncSigsAndDEXsForNetwork() {
 }
 
 # =============================================================================
+<<<<<<< Updated upstream
```

Shellcheck (0.11.0)

[error] Couldn't parse this here string. Fix to allow more checks.

(SC1073)